client/src/pages/Lobby/Lobby.tsx

```diff
@@ -26,6 +27,7 @@ const Lobby: FC<LobbyProps> = ({
   return (
     <div className='Lobby'>
       <>
+        <Invite gameUrl={`${window.location}`} />
```

Comment:
No need to stringify the whole `window.location` object, just pass its `href` property
```suggestion
        <Invite gameUrl={window.location.href} />
```